### ISSUE TITLE
Mouse Cursor Hover missing for Adjustable Frame. 

### ISSUE THREAD

**Issue by cwoneil** [pop-os/pop#1188]
**Distribution (run `cat /etc/os-release`):**

NAME="Pop!_OS"
VERSION="20.04 LTS"
ID=pop
ID_LIKE="ubuntu debian"
PRETTY_NAME="Pop!_OS 20.04 LTS"
VERSION_ID="20.04"
HOME_URL="https://pop.system76.com"
SUPPORT_URL="https://support.system76.com"
BUG_REPORT_URL="https://github.com/pop-os/pop/issues"
PRIVACY_POLICY_URL="https://system76.com/privacy"
VERSION_CODENAME=focal
UBUNTU_CODENAME=focal
LOGO=distributor-logo-pop-os

**Related Application and/or Package Version (run `apt policy $PACKAGE NAME`):**
cinnamon:
  Installed: 4.4.8-4
  Candidate: 4.4.8-4
  Version table:
 *** 4.4.8-4 500
        500 http://us.archive.ubuntu.com/ubuntu focal/universe amd64 Packages
        100 /var/lib/dpkg/status

**Issue/Bug Description:**
Using both Nautilus and Nemo the mouse cursor does not change to the "double arrow" or "dragable cursor" when placed over the bar between the dual panes or between the sidebar and the file list.  It is still possible to drag these bars to adjust the related widths, but you need to guess where to place the mouse due to the lack of visual feedback. 

**Steps to reproduce (if you know):**
This error occurred on an upgrade to pop 20.04 with cinnamon desktop and on different computer with a fresh install of pop 20.04 with cinnamon-desktop.  This error is not present on a recently upgraded ubuntu 20.04 machine. 

**Expected behavior:**
I expect to have the mouse cursor change to indicate that the frame in the window element is draggable and can be moved.  This behaviour is what happens when logging out and back in using the Pop-Desktop window manager and adjusting the frame on either of nautilus or nemo. 

**Other Notes:**
The screenshot shows the frame moved successfully.  The screenshot does not include the mouse cursor, but it stays an arrow through the move. 
![Screenshot from 2020-08-12 23-21-26](https://user-images.githubusercontent.com/69613624/90090773-e0cdc700-dcf2-11ea-8488-044ab1024a8a.png)
![Screenshot from 2020-08-12 23-21-33](https://user-images.githubusercontent.com/69613624/90090775-e1665d80-dcf2-11ea-938f-52ef11fe615b.png)

**Comment by mmstick** [comment]
I cannot replicate this

**Comment by cwoneil** [comment]
As you requested more information, I started to search for cursor settings to see what I could provide.  I found under Menu->Preferences->Themes that the Mouse Pointer theme was set to Adwaita.  I changed the theme to Pop and this solved the problem on the fresh install of Pop-os.  After I changed the theme to Pop, I could not change it back to Adwaita, so I can no longer duplicate the problem on this computer.  

On the second computer which upgraded to Pop 20.04, changing the theme to Pop did not resolve the problem.  However, I ran sudo apt install breeze-cursor-theme and changed to this theme and that did solve the problem.  

Thank you for trying to replicate the problem.  

This issue is resolved for me.
